var/spack/repos/builtin/packages/ecmwf-atlas/package.py

```diff
@@ -20,6 +20,7 @@ class EcmwfAtlas(CMakePackage):
 
     version("master", branch="master")
     version("develop", branch="develop")
+    version("0.36.0", sha256="39bf748aa7b22df80b9791fbb6b4351ed9a9f85587b58fc3225314278a2a68f8")
```

Comment:
Changes to this file will no longer be needed. We merged the missing ecbuild@3.4: dependency upstream (https://github.com/spack/spack/pull/43133); spack develop already had 0.36.0. We'll bring both updates to our spack fork shortly.